test/utility/test_util.i

```diff
@@ -108,8 +111,15 @@ testutil_clean_work_dir(char *dir)
 		testutil_die(ENOMEM, "Failed to allocate memory");
 
 	snprintf(buffer, inputSize, "%s%s", RM_COMMAND, dir);
-	if ((ret = system(buffer)) != 0)
-		testutil_die(ret, "System call to remove directory failed");
+
+	existp = 0;
+	if ( __wt_exist(NULL, dir, &existp) != 0 )
```

Comment:
It would also be normal to save the return value into ```ret```, and use it in the error message.